launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -107,6 +112,33 @@ CSettingsView::CSettingsView(QWidget * parent)
 {
 	ui->setupUi(this);
 
+	// TODO: read available resolutions from mods
```

Comment:
Will be challenging from launcher because it doesn't initialize VLC.
suggestion: use settings.json instead

Reply:
sorry, not sure how settings.json can help determine all supported resolutions...

Reply:
you can move hardcoded values there

Reply:
but settings.json is "user-generated" file, it makes no sense to place hardcoded resolutions there.

Or are you talking about some other config file?

Reply:
What do you think of the additional quick hack (for the time being, of course): add resolutions to the list only when Extras are installed? In launcher that would be a dead simple file presence check at hardcoded location.

Reply:
For settings.json it can have same logic as for other configs. Some default values can be there + mods overrides + user overrides. Separate can be also an option.

And yeah, I agree with your proposal. That's the best one: we have file with single default option and extras can override those values. The only problem is we need to teach launcher to load mods at some level